test/unit/components/navbarpatientcard.test.js

```diff
@@ -6,9 +6,44 @@
 var React = require('react');
 var TestUtils = require('react-addons-test-utils');
 var expect = chai.expect;
+import { shallow } from 'enzyme';
 
 var NavbarPatientCard = require('../../../app/components/navbarpatientcard');
 
+const rootuser = {
+  patient: {
+    permissions: {
+      root: {},
+    },
+  },
+  permsOfLoggedInUser: {
+    note: {},
+    upload: {},
+    view: {},
+  },
+};
+
+const clinicianupload = {
```

Comment:
Also, as above, suggest `careTeamMemberUpload` etc. as opposed to clinician in variable names and test descriptions